rlp/sedes/text.py

```diff
@@ -0,0 +1,54 @@
+from rlp.exceptions import SerializationError, DeserializationError
+from rlp.atomic import Atomic
+
+
+class Text:
+    """A sedes object for encoded text data of certain length.
+
+    :param min_length: the minimal length in encoded characters or `None` for no lower limit
+    :param max_length: the maximal length in encoded characters or `None` for no upper limit
+    :param allow_empty: if true, empty strings are considered valid even if
+                        a minimum length is required otherwise
+    """
+
+    def __init__(self, min_length=None, max_length=None, allow_empty=False, encoding='utf8'):
+        self.min_length = min_length or 0
+        self.max_length = max_length or float('inf')
```

Comment:
Is rlp length encoding really unbounded, or is it `2**256` max length?

Reply:
Truly unbounded (binary is the same).